tsconfig.json

```diff
@@ -17,7 +17,6 @@
       "es6"
     ],
     "types": [
-      "core-js",
```

Comment:
@zigzag95 i think this was a result of the merge with master, as i believe it is not in tsconfig.json in the master branch....however i didnt get any errors removing it, but i am adding it back now